### ISSUE TITLE
Highlighting issue with empty lines

### ISSUE THREAD

**Issue by sbstp** [toroidal-code/language-ocaml#36]
It seems there's an issue with highlighting caused by empty lines

The empty line between Some and None causes anything below it to not be properly highlighted. If you remove it, everything goes back to normal. Obviously this is an issue when you have big type declarations with comments and empty lines to make it more readable.
```ocaml
type 'a option =
    | Some of 'a
    
    | None
  
let x = a
```

**Comment by sbstp** [comment]
adding @AndreasDahl

**Comment by mroch** [comment]
should be fixed by f2980afd6a30adf154bc6ee702bb7b97d678cff3 (in 1.9.2). let me know if not!
